doc/adding_users.rst

```diff
@@ -1,7 +1,13 @@
 Requesting Lab Access
 =====================
 
-Send a lab admin an email with two pieces of information:
+Send a lab admin an email with these pieces of information:
+
+#. Whether you are only requesting access to schedule jobs, or also to run
+   one-off jobs. The latter is currently more restricted.
```

Comment:
"more restricted" meaning harder to get access to it?
I'm not clear on the distinction between these two groupings. Do you mean "access to scheduleD jobs", meaning the ability to read logs, for the first one?